### DISCUSSION TITLE
Checkout Accessory or Consumable to location

### DISCUSSION THREAD

**Discussion by jwasner** [snipe/snipe-it#9849] [Ideas]
Sorry if I missed it elsewhere, but we have a need to checkout accessories or consumables to locations rather than users. I see that https://github.com/snipe/snipe-it/issues/6531 was closed due to low interest. Is this something than anyone else out there in Snipe-IT land could benefit from, or a feature that is possibly coming in v6?

We stock several low-cost/abused/frequently broken items like keyboards, phone handsets, etc. that are shared by multiple users at a location, users who don't (and won't) exist in our Snipe IT instance, hence the need to assign to location.

**Comment by benoithcc** [comment]
Same here!

**Comment by benoithcc** [comment]
https://github.com/snipe/snipe-it/issues/5140

**Comment by kiriaki** [comment]
Definitely need this!

**Comment by psmalley-ati** [comment]
I need this. 
We have common areas (conference rooms and shared desks) that are not assigned to a person, however, we still need to track those accessories used in common.

**Comment by victor-cereijo-idf** [comment]
Instead, we need workplaces to be assignable as users and workplaces to be on an specific location.
But could also do the job as we can use locations as desk/workplaces

**Comment by snakebag** [comment]
I would find it helpful to be able to checkout Consumables and Accessories to Assets as well.  We codify all our routing and access network equipment as Assets and cables and optics as Consumables.  Being able to check out Consumables to Assets or Location would benefit us tremendously.

**Comment by psmalley-ati** [comment]
We need this feature, as well as checking accessories out to an asset.
This has been a need since v4, why are we still needing to beg for it?

**Comment by patricksmalley** [comment]
Any movement on this? I am at a new job and I am using Snipe here too.
They could use this feature as well!

**Comment by jamjarae** [comment]
Just started using Snipe-IT and noticed this limitation as I was loading in consumables, Not all consumables go to users, sometimes it's to an asset or to a location.

**Comment by jackmitchellNU** [comment]
I would find it helpful for this feature to be implemented. It would be better for a majority of our consumables to be checked out to an asset as opposed to a user.
